src/CognitiveServices/CognitiveServices/ChangeLog.md

```diff
@@ -22,4 +22,7 @@
 ## Version 1.0.0
 * General availability of `Az.CognitiveServices` module
 * Add completers for SkuName and Typem available on New-AzureRmCognitiveServicesAccount operation.
-* Removed GetSkusWithAccountParamSetName parameter set from Get-AzCognitiveServicesAccountSkus
\ No newline at end of file
+* Removed GetSkusWithAccountParamSetName parameter set from Get-AzCognitiveServicesAccountSkus
+
+## Version 1.3.0
+* Added CustomSubdomainName as a new optional parameter for New-AzCognitiveServicesAccount which is used to specify subdomain for the resource.
```

Comment:
Please just add this bullet point directly under ## Upcoming Release and remove the version tag you created - we will do this during the release processing.

Reply:
Done.